descartes_ikfast/package.xml

```diff
@@ -3,12 +3,16 @@
   <name>descartes_ikfast</name>
   <version>0.1.0</version>
   <description>The descartes_ikfast package</description>
-
+  <author email="jmeyer1292@github.com">Jon Meyer</author>
```

Comment:
The `author` element does not require the `email` attribute set.

You could leave it empty.

Only `maintainer` must have `email `.